__tests__/__packages__/cli-test-utils/src/environment/TestEnvironment.ts

```diff
@@ -145,9 +145,10 @@ export class TestEnvironment {
             // injectCliProps(properties);
             // console.log(properties);
         } catch (error) {
-            logger.error("Error reading test properties yaml configuration file. Tests cannot continue. " +
-                "Additional details:" + error);
-            throw new Error(error);
+            const errMsg: string = "Error reading test properties yaml configuration file. Tests cannot continue. " +
+                "Additional details:" + error;
+            logger.error(errMsg);
+            throw new Error(errMsg);
```

Comment:
Should we rethrow the original error? `throw error;`

My only concern with the current approach, is that when the `error` object is stringified, some traceback details may be lost.

Reply:
I thought about re-throwing the original error. It would have been easier. I thought I would add the extra text information that was available, but I did not think about the traceback.

I will change to re-throw the error.